src/home.js

```diff
@@ -0,0 +1,74 @@
+import { getMealRecipe } from './comment.js';
+
+import { apiPost, apiGet } from './api.js';
+
+const envolevementUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/5UpYqnub5KIZMG9nlN2D';
+
+const showLikes = (response, span, idMeal) => {
+  const data = response.filter((item) => item.item_id === idMeal);
+  if (data.length !== 0) {
+    span.innerText = `${data[0].likes} likes`;
+  } else {
+    span.innerText = '0 likes';
+  }
+};
+
+const updateLikes = async (span, idMeal) => {
+  apiGet(`${envolevementUrl}/likes`).then((response) => {
+    showLikes(response, span, idMeal);
+  });
+};
+
+const displayDish = (dish, likes) => {
+  const { idMeal, strMeal, strMealThumb } = dish;
+  const card = document.createElement('div');
+  const image = document.createElement('img');
+  const name = document.createElement('h5');
+  const likeB = document.createElement('button');
+  const nameLike = document.createElement('div');
+  const likeSpan = document.createElement('span');
+  const button = document.createElement('button');
+  const reservation = document.createElement('button');
+  card.setAttribute('data-id', idMeal);
+  image.setAttribute('src', strMealThumb);
```

Comment:
- [ ] Please, don't forget to add the `alt` attribute in HTML image elements. It is HTML best practice requirement. :pray: